theano/tensor/tests/test_opt.py

```diff
@@ -3362,25 +3362,19 @@ def test_local_subtensor_of_alloc():
             for slices in slicess:
                 z = yx.__getitem__(slices)
                 f = theano.function([x], z)
-                assert check_stack_trace(f, ops_to_check='all')
                 if theano.config.mode != 'FAST_COMPILE':
                     # Subtensor can be in the input of Alloc
                     assert not isinstance(f.maker.fgraph.toposort()[-1].op,
                                           Subtensor)
+                    if not (isinstance(f.maker.fgraph.toposort()[-1].op,
+                                       DeepCopyOp) and len(f.maker.fgraph.toposort())==1):
```

Comment:
I think you can remove that check, passing `bug_print='ignore'` should be enough.